--- cuyo-1.8.2/src/Makefile.am~	2003-09-29 20:38:22.000000000 +0000
+++ cuyo-1.8.2/src/Makefile.am	2003-09-29 20:39:19.000000000 +0000
@@ -65,7 +65,7 @@
 ## Dadurch, dass es so hier steht, kann man das
 ## Data-Dir zu Makezeiten noch ändern (im Gegensatz zu: steht in configure.in;
 ## dann ist es nach ./configure schon festgelegt).
-CXXFLAGS = $(QT_CXXFLAGS) -DPKGDATADIR=\"$(pkgdatadir)\" -Wall -W -ansi -pedantic $(CXXDBGFLAGS) @CXXFLAGS@
+CXXFLAGS = $(QT_CXXFLAGS) -DPKGDATADIR=\"$(pkgdatadir)\" -Wall -W -ansi $(CXXDBGFLAGS) @CXXFLAGS@
 
 
 # Moc-Dateien erzeugen (QT_MOC wird von BNV_HAVE_QT gesetzt)
